rasa/version.py

```diff
@@ -1,3 +1,3 @@
 # this file will automatically be changed,
 # do not add anything but the version number here!
-__version__ = "3.6.4"
+__version__ = "3.7.0a1"
```

Comment:
Are you sure this needs to be done? We also have 3.7.0b1, 3.7.0b2 and 3.8.0 releases too but they didn't update here

Reply:
That was done automatically by the `make release` command.
All I did was change the rasa-sdk version.

Reply:
> We also have 3.7.0b1, 3.7.0b2 and 3.8.0 releases too but they didn't update here

They did update this file, but then I merged 3.6.x into main and that's why the latest version now recorded is `3.6.4`.